cmd/podman/ps.go

```diff
@@ -414,6 +414,8 @@ func getTemplateOutput(containers []*libpod.Container, opts psOptions) ([]psTemp
 			status = "Paused"
 		case libpod.ContainerStateCreated:
 			status = "Created"
+		case libpod.ContainerStateConfigured:
+			status = "Created"
```

Comment:
should this be Configured?

Reply:
nope, Created per the agreement on irc

Reply:
Can you change the code to 
case libpod.ContainerStateCreated, libpod.ContainerStateConfigured:
              status = "Created" 

So future engineers will know this is intentional.

Reply:
For context, so we have a record of the reason we did this: the distinction between Created and Configured is very non-obvious to users, and we're trying to hide it as much as possible at the podman side (I'd actually like to start hiding it in libpod too)

Reply:
did what @rhatdan  asked.  nice shortcut!